arabic/unpacked/html.js

```diff
@@ -0,0 +1,90 @@
+MathJax.Hub.Config({
+  'HTML-CSS': {
+    undefinedFamily: 'Amiri',
```

Comment:
I might just make it a suggestion in the docs and remove this settings altogether from the extension itself.